plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergMaterializedViews.java

```diff
@@ -113,13 +115,15 @@ public void testMaterializedViewsMetadata()
 
         // test freshness update
         assertQuery(
-                format("SELECT is_fresh FROM system.metadata.materialized_views WHERE name = '%s'", materializedViewName),
+                // TODO (https://github.com/trinodb/trino/issues/9039) remove redundant schema_name filter
+                format("SELECT is_fresh FROM system.metadata.materialized_views WHERE schema_name = '%s' AND name = '%s'", schemaName, materializedViewName),
```

Comment:
What is the other test that is causing race condition? 
Since there seems to be race condition, why we only expect single result?

Reply:
we expect single result becaue we select a single row from system.metadata.materialized_views, using randomized materializedViewName

Reply:
What is the other test that is causing race condition?

Reply:
idk, but  see https://github.com/trinodb/trino/pull/9042#issuecomment-908606625 for repro steps